src/content/0/en/part0b.md

```diff
@@ -303,7 +303,7 @@ The same treelike structure can be seen on the console tab <i>Elements</i>.
 
 The functioning of the browser is based on the idea of depicting HTML elements as a tree. 
 
-Document Object Model, or [DOM](https://en.wikipedia.org/wiki/Document_Object_Model) is an Application Programming Interface, (an <i>API</i>), which enables programmatic modification of the <i>element trees</i> corresponding to web-pages.
+Document Object Model, or [DOM](https://en.wikipedia.org/wiki/Document_Object_Model), is an Application Programming Interface (an <i>API</i>), which enables programmatic modification of the <i>element trees</i> corresponding to web-pages.
```

Comment:
I might be totally wrong about the comma placing here but the comma in front of "(an API)" felt wrong while reading.